[2022-04-19 14:15:23,448] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hook_dag.log_end_no_data_1 manual__2022-04-19T14:15:15.629326+00:00 [queued]>
[2022-04-19 14:15:23,460] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hook_dag.log_end_no_data_1 manual__2022-04-19T14:15:15.629326+00:00 [queued]>
[2022-04-19 14:15:23,460] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-19 14:15:23,460] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-04-19 14:15:23,460] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-19 14:15:23,471] {taskinstance.py:1264} INFO - Executing <Task(PostgresOperator): log_end_no_data_1> on 2022-04-19 14:15:15.629326+00:00
[2022-04-19 14:15:23,477] {standard_task_runner.py:52} INFO - Started process 16372 to run task
[2022-04-19 14:15:23,479] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hook_dag', 'log_end_no_data_1', 'manual__2022-04-19T14:15:15.629326+00:00', '--job-id', '494', '--raw', '--subdir', 'DAGS_FOLDER/hook_dag.py', '--cfg-path', '/tmp/tmp1vblbmyx', '--error-file', '/tmp/tmpkdmwxo46']
[2022-04-19 14:15:23,479] {standard_task_runner.py:77} INFO - Job 494: Subtask log_end_no_data_1
[2022-04-19 14:15:23,535] {logging_mixin.py:109} INFO - Running <TaskInstance: hook_dag.log_end_no_data_1 manual__2022-04-19T14:15:15.629326+00:00 [running]> on host 22e04adb6bfa
[2022-04-19 14:15:23,586] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hook_dag
AIRFLOW_CTX_TASK_ID=log_end_no_data_1
AIRFLOW_CTX_EXECUTION_DATE=2022-04-19T14:15:15.629326+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-19T14:15:15.629326+00:00
[2022-04-19 14:15:23,593] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-19 14:15:23,597] {dbapi.py:225} INFO - Running statement: UPDATE list_flow_log
SET q_source=null, q_target=null, t_end=NOW(), status=2, optional_key=null, comment = 'not_new_data'
where id = 27;
, parameters: None
[2022-04-19 14:15:23,598] {dbapi.py:233} INFO - Rows affected: 1
[2022-04-19 14:15:23,608] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=hook_dag, task_id=log_end_no_data_1, execution_date=20220419T141515, start_date=20220419T141523, end_date=20220419T141523
[2022-04-19 14:15:23,655] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-19 14:15:23,684] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
